crates/telio-relay/src/derp/http.rs

```diff
@@ -154,6 +202,7 @@ async fn connect_and_start<RW: AsyncRead + AsyncWrite + Send + 'static>(
     secret_key: SecretKey,
     server_keepalives: DerpKeepaliveConfig,
     host: &str,
+    derp_v2: bool,
```

Comment:
I suggest using an enum in this case, boolena is not very descriptive when speaking about versions and it also makes lookup more difficult. Having something like `DerpVersion::V2` being passed would make it very clear what this parameter is about